### DISCUSSION TITLE
Please can you help in a testing

### DISCUSSION THREAD

**Discussion by KendallDoesCoding** [TechStudent10/techstudent10#29] [General]
Hi,

I added a [greeting workflow](https://github.com/KendallDoesCoding/languages-around-the-world/blob/main/.github/workflows/greetings.yml) to my repo and tried testing on my alt.. Although my alt is flagged and has been flagged since I created the account (contacted support today), so please make a issue or PR for testing in this repo: https://github.com/KendallDoesCoding/languages-around-the-world

**Comment by TechStudent10** [comment]
Works

![image](https://user-images.githubusercontent.com/76978184/155411693-43f25605-9008-48ca-bd3a-77a1ec558bd0.png)

**Comment by KendallDoesCoding** [comment]
Yup, just saw that...  Thanks for making the issue, I tried to same on my
alt, but as I said my account is flagged so it didn't work.. So ig only
requirement for it to work, is not to have a account flagged, which more
than half of the users don't have...
